docs/annotations.md

```diff
@@ -0,0 +1,229 @@
+# Annotation Schema
+
+The full schema can be obtained by calling the Girder endpoint of `GET` `/annotation/schema`.
+
+An annotation consists of a basic structure which includes a free-form `attributes` object and a list of `elements`.  The elements are strictly specified by the schema and are limited to a set of defined shapes.
+
+Partial annotations are shown below with some example values.  Note that the comments are not part of a valid annotation:
+
+```
+{
+  "name": "MyAnnotationName",              # Non-empty string.  Optional
+  "description": "This is a description",  # String.  Optional
+  "attributes": {                          # Object.  Optional
+    "key1": "value1",
+    "key2": ["any", {"value": "can"}, "go", "here"] 
+  },
+  "elements": []                           # A list.  Optional.  
+                                           # See below for valid elements.
+}
+```
+
+## Elements
+
+Currently, all defined elements are shapes.  All of these have some properties that they are allowed.  Each shape is listed below:
+
+### All shapes
+
+All shapes have the following properties.  If a property is not listed, it is not allowed.  If element IDs are specified, they must be unique.
+
+```
+{
+    "type": "point",                  # Exact string for the specific shape.  Required
+    "id": "0123456789abcdef01234567", # String, 24 lowercase hexadecimal digits.  Optional.
+    "label": {                        # Object.  Optional
+      "value": "This is a label",     # String.  Optional
+      "visability": "hidden",         # String.  One of "always", "hidden", "onhover".  Optional
+      "fontSize": 3.4,                # Number.  Optional
+      "color": "#0000FF"              # String.  See note about colors.  Optional
```

Comment:
Font color (though we might have to as @brianhelba to be certain).  I don't think that we currently have a concept of a label bound box and background (or even relative positioning).